CHANGELOG.md

```
@@ -1,39 +1,38 @@
 # Prowide ISO 20022 - CHANGELOG
 
-### 10.2.8 - SNAPSHOT
+#### 9.5.8 - SNAPSHOT
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)